ddl/db_change_test.go

```diff
@@ -673,6 +684,14 @@ func (s *testStateChangeSuite) testControlParallelExecSQL(c *C, sql1, sql2 strin
 	c.Assert(err, IsNil)
 	defer s.se.Execute(context.Background(), "drop table t")
 
+	_, err = s.se.Execute(context.Background(), "drop database if exists t_part")
```

Comment:
I think we can add an argument to distinguish between using a partitioned table or a general table.

Reply:
@zimulala master branch has opened the range partition table by default.

Reply:
@zimulala PTAL.